### PR TITLE
OS:  Add initial support for IRIX

### LINKED ISSUE #706: Support for IRIX
## Description
Running $ neofetch on SGI IRIX 6.5.29 (on an SGI Octane) returns the following:
Unknown OS detected: 'IRIX64', aborting...
Open an issue on GitHub to add support for your OS.

I am happy to provide any information, support, testing, etc; to add IRIX to the list of support UNIX systems (as it is a true, certified UNIX).

$ uname -a
IRIX64 Octane 6.5 01090133 IP30

#### Neofetch version
3.1.0

### PR DESCRIPTION
**Pull request by dylanaraps**
## Description

Title.

## TODO

I've excluded functions that only require an X server or work on all OS already.

- [x] `get_os()`
- [x] `get_distro()`
- [ ] `get_model()`
    - [Use an iPhone-like method?](http://www.tek-tips.com/viewthread.cfm?qid=1075647)
- [x] `get_kernel()`
    - Disabled function
- [x] `get_uptime()`
- [x] `get_packages()`
- [x] `get_cpu()`
- [x] `get_gpu()`
    - <s>Is this necessary?</s>Superseded by PR #711 
- [x] `get_memory()`
- [x] `get_disk()`
    - Workaround used since their `df` doesn't support `-h`.
- [x] <s>`get_battery()`</s>
    - Unneeded. [There are no IRIX laptops](http://www.siliconbunny.com/silicon-graphics-laptops/).
- [x] `get_local_ip()`
    - <s>We need example `netstat -ia` output.</s>The good ol' `ifconfig` will do.
- [x] `get_install_date()`
    - Disabled their `ls` and used `/`.

## Note

All of this is untested, I'm going off of google search results.

Closes #706

### CODE REVIEW

**Comment by konimex** [comment]
I need to rework on `get_cpu` because apparently they have different outputs for MHz and GHz. This can be tricky.

File: neofetch [hunk 1]
```diff
@@ -317,8 +322,8 @@ get_title() {
 }
 
 get_kernel() {
-    # Since AIX is an integrated system, it's better to skip this function altogether
-    [[ "$os" == "AIX" ]] && return
+    # Since these OSes are integrated systems, it's better to skip this function altogether
```
Comment (dylanaraps) [inline neofetch]: "These OS are" sounds better.

File: neofetch [hunk 2]
```diff
@@ -971,6 +980,17 @@ get_cpu() {
                 "physical") cores="$(lparstat -i | awk -F':' '/Active Physical CPUs/ {printf $2}')"
             esac
         ;;
+
+        "IRIX")
+            # Get CPU name.
+            cpu="$(hinv -c processor | awk -F':' '/CPU:/ {printf $2}')"
+
+            # Get CPU speed.
+            cpu="$(hinv -c processor | awk '/MHZ/ {printf $2}')"
```
Comment (dylanaraps) [inline neofetch]: `speed=`

(You're setting speed as CPU name)

File: neofetch [hunk 3]
```diff
@@ -1810,7 +1839,15 @@ get_disk() {
         disk_info=($disk)
         disk_perc="${disk_info[4]/'%'}"
 
-        disk="${disk_info[2]/i} / ${disk_info[1]/i} (${disk_perc}%)"
+        case "$df_version" in
+            *"befhikm"*)
+                disk="$((disk_info[2]/1024/1024))G / $((disk_info[2]/1024/1024))G (${disk_perc}%)"
+            ;;
+
+            *)
```
Comment (dylanaraps) [inline neofetch]:
```sh
*) disk="${disk_info[2]/i} / ${disk_info[1]/i} (${disk_perc}%)" ;;
```

**Comment by konimex** [comment]
As of now this branch is ready to be merged into master.

readymerge

The `bash` before `./neofetch` is only needed because the bash version shipped in the machine is bash-2.05b.0, otherwise it runs fine.

**Comment by dylanaraps** [comment]
LGTM :+1:

I'll also flag a new release tomorrow if that's ok. :)

**Pull request merged**